<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Kugan-PC" name="Test" tests="1" failures="1" timestamp="4 Aug 2020 10:50:57 GMT" time="103.056" errors="0">
  <testcase name="vali_Text" time="1.66" classname="completeFramework.ValidateText">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at completeFramework.ValidateText.vali_Text(ValidateText.java:33)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- vali_Text -->
</testsuite> <!-- Test -->
